### ISSUE TITLE
Viz not recover after error

### ISSUE THREAD

**Issue by mstefaniuk** [mdaines/viz.js#59]
When Viz is called subsequently (the same instance) it cannot recover when unclosed quotes error is thrown. Failing case looks as follows:

``` javascript
define(["viz"], function(viz) {

  describe("Viz.js", function() {
    it("should recover after unclosed quotes error", function() {
      expect(function() {
        viz([
          'digraph {',
          ' a -> b [label="erroneous]',
          '}'].join("\n"));
      }).toThrow();
      var svg = viz([
        'digraph {',
        ' a -> b [label="correcteous"]',
        '}'].join("\n"));
      expect(svg).toBeDefined();
    });
  });
});
```

First call throws an exception with `Error: syntax error in line 2 near ''` as expected. Subsequent gives `Error: syntax error in line 1 near '"'` until new Viz instance is created.

Viz is loaded using `require.js` with shim.

**Comment by jackycute** [comment]
We also experienced this issue.

**Comment by mdaines** [comment]
c687a63 should address this. I am attaching a build of that version if you wish to test it.

[viz.js.zip](https://github.com/mdaines/viz.js/files/682970/viz.js.zip)

**Comment by jackycute** [comment]
I just tested it, it works.
But the error message seems a bit weird.
```
abort() at Error
If this abort() is unexpected, build with -s ASSERTIONS=1 which can give more information.
```

**Comment by mdaines** [comment]
So it does. I'll update the test to check for the correct error message and see if I can find the problem.

**Comment by mdaines** [comment]
This should be addressed in 2257a4017ed3bdc4bbf635279d9cd066091a425b and the fix will be in the next release.
